test/DebugInfo/DumpTypeFromMangledName.swift

```diff
@@ -0,0 +1,26 @@
+// RUN: %empty-directory(%t)
+
+// RUN: %target-build-swift -emit-executable %s -g -o %t/ASTSection -emit-module
+
+// RUN: echo '$S4blah4mainyyF' >> %t/input.sym
+// RUN: echo '$SytD' >> %t/input.sym
+// RUN: echo '$Ss5Int32VD' >> %t/input.sym
+// RUN: echo '$S4blah4mainyyF8PatatinoL_VMa' >> %t/input.sym
+
+// RUN: %lldb-moduleimport-test %t/ASTSection \
+// RUN:   -type-from-mangled='%t/input.sym' 2>&1 | %FileCheck %s
+
+// REQUIRES: executable_test
+func main() {
+  struct Patatino {}
+}
+
+// CHECK: Type of $S4blah4mainyyF
```

Comment:
Perhaps interleave the RUN commands with the CHECKs so they are grouped together?